src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -69,6 +70,9 @@ public Command parseCommand(String userInput) throws ParseException {
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+            if (!arguments.isEmpty()) {
+                throw new ParseException(LIST_MESSAGE_INVALID_COMMAND);
+            }
```

Comment:
Would it be better to have this inside a ListCommandParser() for better SLAP? I think it would help improve consistency between all the different methods of parsing user commands! :)

Reply:
I will create a seperate issue and PR for the documentation update.